src/verifiable/keys.ts

```diff
@@ -1,100 +1,44 @@
-import { promises } from 'fs'
+import fromKeyLike from 'jose/jwk/from_key_like'
+import calculateThumbprint from 'jose/jwk/thumbprint'
+import generateKeyPair from 'jose/util/generate_key_pair'
+import { JWK } from 'jose/webcrypto/types'
 
-import { JWK, JWKECKey, JWKOctKey, JWKOKPKey, JWKRSAKey, JWS } from 'jose'
-
-import RSAKey = JWK.RSAKey
-import ECKey = JWK.ECKey
-import OKPKey = JWK.OKPKey
-import OctKey = JWK.OctKey
-import JWSRecipient = JWS.JWSRecipient
+import {
+  ProtectedHeaders,
+  VerifiedAddressSignature,
+} from './verifiable-paystring'
 
+const DEFAULT_ALGORITHM = 'ES256'
 const DEFAULT_CURVE = 'P-256'
 
-/**
- * Reads JWK key from a file.
- *
- * @param path - The full file path of the key file.
- * @returns A JWK key.
- */
-export async function getSigningKeyFromFile(
```

Comment:
removed file related functions that were only used by the CLI. These functions will be added to the CLI project.